### ISSUE TITLE
model.load_model()

### ISSUE THREAD

**Issue by JohnLoveMm** [cyprienruffino/CTCModel#14]
I want know  can i save the model?when if trained  the CTCModel,i want keep the model file,but there rise an error,somebody could help me?(I know the reason,it is because there no save module in CTCMolel)

**Comment by JohnLoveMm** [comment]
today,I find the save_model function,but,when I load the model,it can not predict,who can help me

**Comment by cyprienruffino** [comment]
Hi ! Can you please post the error messages you're getting ?

**Comment by JohnLoveMm** [comment]
sorry for waiting so long,I use this "netsork.save_model(path)" saved the model successfuly,but when I load the model, I use "CTCModel.load_model(path)",there rise the error"load_model() missing 2 required positional arguments: 'path_dir' and 'optimizer'
",In fact I do not know how to load my model.
could I keep your wechat?Because I have so much problem about CTCModel.
Finally,Thank you very much for answer my question.

**Comment by JohnLoveMm** [comment]
> Hi ! Can you please post the error messages you're getting ?

sorry for waiting so long,I use this "netsork.save_model(path)" saved the model successfuly,but when I load the model, I use "CTCModel.load_model(path)",there rise the error"load_model() missing 2 required positional arguments: 'path_dir' and 'optimizer'
",In fact I do not know how to load my model.
could I keep your wechat?Because I have so much problem about CTCModel.
Finally,Thank you very much for answer my question.

**Comment by JohnLoveMm** [comment]
> Hi ! Can you please post the error messages you're getting ?

when i create a new network,that did not trained,
`MODEL = create_network(nb_features, nb_labels, padding_value)
model = MODEL.load_model(path, Adam)
`
there rise a new error"Could not interpret optimizer identifier: <class 'keras.optimizers.Adam'>"
I use the same optimizer,in train and load model,I changed "tensorflow.keras.optimizers.Adam" to "keras.optimizers.Adam",but neither worked.In the train cycle I use "keras.optimizers.Adam".I do not know how to solve it,I hope you could help me.

**Comment by JohnLoveMm** [comment]
> > Hi ! Can you please post the error messages you're getting ?
> 
> when i create a new network,that did not trained,
> `MODEL = create_network(nb_features, nb_labels, padding_value) model = MODEL.load_model(path, Adam) `
> there rise a new error"Could not interpret optimizer identifier: <class 'keras.optimizers.Adam'>"
> I use the same optimizer,in train and load model,I changed "tensorflow.keras.optimizers.Adam" to "keras.optimizers.Adam",but neither worked.In the train cycle I use "keras.optimizers.Adam".I do not know how to solve it,I hope you could help me.

Oh,yes, I know,there some thing wrong with my setting.i should use the same optimizer with the same learning rate,it will be work in this:`model = MODEL.load_model(path, Adam(lr=0.001))`

**Comment by JohnLoveMm** [comment]
> Hi ! Can you please post the error messages you're getting ?

could you please give an example on how to save a model,and how to load a trained model? I can not do this,it is so hard for me

**Comment by cyprienruffino** [comment]
Hi ! I'll update the readme with an example. Thanks for the report !
